### ISSUE TITLE
mpv window size doesn't match video resolution

### ISSUE THREAD

**Issue by neodyne** [mpv-player/mpv#7793]
### Important Information

Provide following Information:
- mpv version
mpv 0.32.0-526-g102a083171 Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
 built on Sun May 31 02:29:11 +08 2020
FFmpeg library versions:
   libavutil       56.49.100
   libavcodec      58.90.100
   libavformat     58.44.100
   libswscale      5.6.101
   libavfilter     7.84.100
   libswresample   3.6.100
FFmpeg version: git-2020-05-31-4aa07d1a

- Windows Version
Windows 10 x64 version 2004 (build 19041.264)
- Source of the mpv binary
https://sourceforge.net/projects/mpv-player-windows/files/64bit/

- If known which version of mpv introduced the problem
mpv 0.32.0-456-g4e94b2177a Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
 built on Sun May 10 23:34:45 +08 2020
FFmpeg library versions:
   libavutil       56.43.100
   libavcodec      58.83.100
   libavformat     58.43.100
   libswscale      5.6.101
   libavfilter     7.80.100
   libswresample   3.6.100
FFmpeg version: git-2020-05-10-5727b1f13
https://sourceforge.net/projects/mpv-player-windows/files/64bit/mpv-x86_64-20200510-git-4e94b21.7z/download

### Reproduction steps
- Open any video file in mpv, not in fullscreen, **without any additional parameters**, with no config. More noticeable with `--no-border`

### Expected behavior
- The opened window matches the resolution of the video file (height/width)

### Actual behavior
- The window is bigger, and the video is stretched to fill the window

### Log file
https://0x0.st/iOHh.txt

The log file is from playing a 480p file (720x404), which gets stretched to 1080x607. Relevant lines:

`[   0.241][v][vo/gpu/win32] reset window bounds: 420:206:1080:607`
`[   0.242][d][vo/gpu/win32] move window: 420:206`
`[   0.242][v][vo/gpu/win32] resize window: 1080:607`
`[   0.247][v][vo/gpu/win32] resize window: 1080:607`
`[   0.247][d][vo/gpu/win32] move window: 420:206`
`[   0.249][v][vo/gpu] Resize: 1080x607`
`[   0.249][v][vo/gpu] Window size: 1080x607 (Borders: l=0 t=0 r=0 b=0)`
`[   0.249][v][vo/gpu] Video source: 720x404 (404:405)`
`[   0.249][v][vo/gpu] Video display: (0, 0) 720x404 -> (0, 0) 1080x607`
`[   0.249][v][vo/gpu] Video scale: 1.500000/1.502475`
`[   0.249][v][vo/gpu] OSD borders: l=0 t=0 r=0 b=0`
`[   0.249][v][vo/gpu] Video borders: l=0 t=0 r=0 b=0`
`[   0.249][v][vo/gpu] Reported display depth: 8`
`[   0.249][v][vo/gpu] Texture for plane 0: 720x404`
`[   0.249][v][vo/gpu] Texture for plane 1: 360x202`
`[   0.249][v][vo/gpu] Texture for plane 2: 360x202`
`[   0.249][v][vo/gpu] Testing FBO format rgba16hf`
`[   0.249][d][vo/gpu] Resizing texture: 16x16`
`[   0.249][v][vo/gpu] Using FBO format rgba16hf.`
`[   0.249][v][vo/gpu] No advanced processing required. Enabling dumb mode.`
`[   0.251][v][vo/gpu] Resize: 1080x607`
`[   0.251][v][vo/gpu] Window size: 1080x607 (Borders: l=0 t=0 r=0 b=0)`
`[   0.251][v][vo/gpu] Video source: 720x404 (404:405)`
`[   0.251][v][vo/gpu] Video display: (0, 0) 720x404 -> (0, 0) 1080x607`

**Comment by CounterPillow** [comment]
And you're not using some sort of busted frontend like SMPlayer, right? Because your log file contains a suspiciously named path.

**Comment by neodyne** [comment]
I mean, the mpv directory is placed there, I usually use SMPlayer as frontend, but this happens when the file is opened directly in **mpv.exe**. To test, I unpacked the mpv .7z from sourceforge to different, fresh directory, no config, no nothing, and the results are the same. Funny enough, it does not happen when the file is played through SMPlayer, but that is probably irelevant.

It seems to me that mpv defaults to `--window-scale=1.5` for no particular reason, but even if I set the scaling to 1.0 it does not have any effect.
